src/index.ts

```diff
@@ -1,23 +1,4 @@
-import { AbiCache } from './abi-cache/abi-cache'
-import { initCache, CacheType } from './abi-cache/cache'
-import { cacheCandidates } from './abi-cache/cache-candidates'
-import { CallDataInformation } from './parser'
-import { parseCallData } from './parser/calldata-parsers/parse-call-data'
-
-export { CacheType } from './abi-cache/cache'
-export { CallDataInformation } from './parser'
-export const parseCallDataString = async (
-  callDataString: string,
-  cache: AbiCache = initCache(CacheType.MEMORY)
-): Promise<CallDataInformation[]> => {
-  const parsedCandidates: CallDataInformation[] = parseCallData(
-    callDataString,
-    cache
-  )
-
-  await Promise.all(
-    parsedCandidates.map((candidate) => cacheCandidates(cache, candidate))
-  )
-
-  return parseCallData(callDataString, cache)
-}
+export * from './abi-cache'
+export * from './chains'
+export * from './parse-call-data-string'
+export * from './parser'
```

Comment:
But this exports everything from our package. My intention was to only export what is really needed.

Reply:
it's not actually everything, but I've checked what we're using on the BE, and we can remove one of these imports. Updated.